swap/swap.go

```diff
@@ -78,9 +80,34 @@ func NewParams() *Params {
 	}
 }
 
+// NewLogger returns a new logger
+func NewLogger(h interface{}) log.Logger {
+	logLevel := 3
```

Comment:
do we want this hardcoded?

Reply:
It could be parametrized.
Not sure where we could load this config.

Reply:
ok, can i ask where the value `3` came from then?

Reply:
`3` is the default verbosity (`internal/debug/flags.go:42`). Perhaps we can let this log-level to be the same as the value of this verbosity: by default, it will be three, but we have to check whether this value was not changed by the user. 

I don't know whether it will be non-trivial though to get how this value is initialized.

Reply:
Fixed,
I left a swapLogLevel const.
The level could be different from the swarm logger.

Reply:
this has been extracted to a `const`, which is good enough for me. resolving.